test/python/test_pack_e2e.py

```diff
@@ -0,0 +1,197 @@
+import pytest
+import os
+import tarfile
+import subprocess
+
+from utils import basepath
+from utils import examine_application_instance_container
+from utils import tarantool_enterprise_is_used
+from utils import Image, find_image, delete_image
+from utils import Archive, find_archive
+from utils import tarantool_repo_version
+
+
+# #######################
+# Class InstanceContainer
+# #######################
+class InstanceContainer:
+    def __init__(self, container, instance_name, http_port, advertise_port):
+        self.container = container
+        self.instance_name = instance_name
+        self.http_port = http_port
+        self.advertise_port = advertise_port
+
+
+# ########
+# Fixtures
+# ########
+@pytest.fixture(scope="function")
+def image_name_for_tests(docker_client, tmpdir, request):
+    if tarantool_enterprise_is_used():
+        return 'centos:8'
+
+    build_path = os.path.join(tmpdir, 'build_image')
+    os.makedirs(build_path)
+
+    test_image_dockerfile_path = os.path.join(build_path, 'Dockerfile')
+    with open(test_image_dockerfile_path, 'w') as f:
+        f.write('''
+            FROM centos:8
+            RUN curl -s \
+                https://packagecloud.io/install/repositories/tarantool/{}/script.rpm.sh | bash \
+                && yum -y install tarantool tarantool-devel
+        '''.format(tarantool_repo_version()))
+
+    IMAGE_NAME = 'test-image'
+    docker_client.images.build(
```

Comment:
Does pydocker support options as "enable build kit" and something for more aggressive caching?

Reply:
No. But it's possible that setting environment variable `DOCKER_BUILDKIT` affects using this feature.